v4.9.1

mikaeldui opened the pull request:
Seems to work fine, @minosimo & @metsma 🙂

DigiDoc4-Client release: [4.9.0](https://github.com/open-eid/DigiDoc4-Client/releases/tag/v4.9.0), [4.9.1](https://github.com/open-eid/DigiDoc4-Client/releases/tag/v4.9.1)
libdigidocpp release: [4.3.0](https://github.com/open-eid/libdigidocpp/releases/tag/v4.3.0)

### Updated dependencies
- flatbuffers 25.2.10 → [25.12.19](https://github.com/google/flatbuffers/releases/tag/v25.12.19)
- xmlsec 1.3.7 → [1.3.9](https://github.com/lsh123/xmlsec/releases/tag/1.3.9)
  - Removed upstreamed patches:
    - ecdsa-sig → https://github.com/lsh123/xmlsec/commit/5a4eb6c10cb27718bc3bbe59998efaca83294cd9
    - rsapss → https://github.com/lsh123/xmlsec/commit/a7e8464f2a2826820b94cc641ac0aae345641fc6
- PCSC 2.3.3 → [2.4.1](https://github.com/LudovicRousseau/PCSC/releases/tag/2.4.1)
  - Removed upstreamed patches:
    - negotiate_protocol → https://github.com/LudovicRousseau/PCSC/commit/db459f95b855ff5a4bd03e46aebad0eac6414cc5
- shared-modules https://github.com/flathub/shared-modules/commit/ec91811777c655b6a3091d60f1eadd79e0fbc957 → https://github.com/flathub/shared-modules/commit/8d705c873c8b04cc2805e0819e2801876fb5cfef
  - libusb 1.0.27 → [1.0.29](https://github.com/libusb/libusb/releases/tag/v1.0.29)

### Tested
- Opening signed .asice
- Signing an .asice with Mobile ID

### Screenshot
<img width="2094" height="1320" alt="image" src="https://github.com/user-attachments/assets/195153b0-0924-4a2a-8a89-b9f814e2985d" />

mikaeldui committed:
v4.9.0

mikaeldui commented on ee.ria.qdigidoc4.yml:
```diff
@@ -35,27 +35,25 @@ modules:
-        url: https://github.com/lsh123/xmlsec/releases/download/1.3.7/xmlsec1-1.3.7.tar.gz
-        sha256: d82e93b69b8aa205a616b62917a269322bf63a3eaafb3775014e61752b2013ea
+        url: https://github.com/lsh123/xmlsec/archive/refs/tags/1.3.9.tar.gz
+        sha256: 534f50743b2fa3ebf3c05917a293c73f255c823dd0285f02622260084317c605
```
This accidental change doesn't seem to be causing any issues. According to the readme you need to run `./autogen.sh` if you build "from GitHub" but I guess `buildsystem: autotools` takes care of that?

mikaeldui committed:
v4.9.1

minosimo commented:
Thanks as always, I will check the ID card and pcsc stuff once I have my reader again!

mikaeldui committed:
Updated dependencies

- flatbuffers → [25.12.19](https://github.com/google/flatbuffers/releases/tag/v25.12.19)
- PCSC → [2.4.1](https://github.com/LudovicRousseau/PCSC/releases/tag/2.4.1)
  - Removed upstreamed patches:
    - negotiate_protocol → https://github.com/LudovicRousseau/PCSC/commit/

mikaeldui committed:
Updated hashes

minosimo committed:
Update SHA256 checksum

mikaeldui committed:
Fix all checksums 😝

minosimo committed:
Updated date to today so it shows in flathub
